### PR TITLE
Added @mapbox/mapbox-gl-rtl-text plugin

### PR DESCRIPTION
**Pull request by orangemug**
`MapboxGl.setRTLTextPlugin` has a odd API where you need to load the plugin via AJAX hence the `copy-webpack-plugin` is required, fixes issue #126

**NOTE:** Adding this plugin now prevents us from running Maputnik from the `file:///` protocol, because the AJAX will fail with

```
Cross origin requests are only supported for protocol schemes: http, data, chrome, chrome-extension, https.
```

This unfortunately makes Maputnik less portable, but I don't think we really have a choice...

### CODE REVIEW

**Comment by lukasmartinelli** [comment]
> NOTE: Adding this plugin now prevents us from running Maputnik from the file:/// protocol, because the AJAX will fail with

Hmmmm. For me this is one of the coolest feature that you can download a ZIP bundle from the releases page and just click on the index.html locally.

I know the work that went into making this possible is huge. But wonder whether it justifies breaking simple local deployment.

To make it more portable can we perhaps point the rtl-plugin to the Mapbox HTTPS url of the plugin?
If you don't have internet it will then perhaps just fail gracefully in the background.

While I really prefer having everything locally that could be a tradeoff to make so you can still just download and open the ZIP file.

**Comment by lukasmartinelli** [comment]
> That's a valid point. Maybe on this one (and only one) occasion we can reach in a mess with the private API (see https://github.com/mapbox/mapbox-gl-js/blob/171db6177a6a5e7246fcef4d74ed297ee5a196d8/src/source/rtl_text_plugin.js#L31-L34)

The inspect plugin also messes with the private API :scream: :wink:

**Comment by orangemug** [comment]
There you go @lukasmartinelli thats a hack for ya! No private APIs required and works on `http://` and `file:///`

File: src/libs/mapbox-rtl.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import MapboxGl from 'mapbox-gl/dist/mapbox-gl.js'
+
+// Load mapbox-gl-rtl-text using object urls without needing http://localhost for AJAX.
+const data = require("base64?mimetype=text/javascript!@mapbox/mapbox-gl-rtl-text/mapbox-gl-rtl-text.js");
```
Comment (lukasmartinelli) [inline src/libs/mapbox-rtl.js]: You smart 😎

**Pull request merged**